sql/dd/impl/bootstrap/bootstrapper.cc

```diff
@@ -1673,6 +1676,25 @@ bool sync_meta_data(THD *thd) {
   return false;
 }
 
+// Helper guard used in update_properties, to be sure
+// that encryption will get set back before
+// update_properties exits.
+struct Target_encryption_guard {
+ public:
+  Target_encryption_guard(const Object_table *object_table)
+      : m_object_table(object_table)
+      : set_encryption(object_table->is_target_encrypted) {}
+  ~Target_encryption_guard() {
+    if (set_encryption) {
+      m_object_table->set_target_encrypted();
+    }
+  }
+
+ private:
+  const Object_table *m_object_table;
+  bool set_encryption{false};
```

Comment:
we can remove this initialization of flase here because now it is initialized at constructor

Reply:
done